client/.storybook/preview-body.html

```diff
@@ -0,0 +1,22 @@
+<style>
```

Comment:
Missing some tags indicating that this is HTML, e.g.:
```
<!DOCTYPE html>
<html>
<head>
…
<title>TBD</title>
</head>
<body>
…
</body>
</html>
```

Reply:
I think `preview-body.html` has some special functionality under the hood so that storybook smartly takes what's inside of this file and put it inside of the body of an html file which we don't actually interact with. I'm not 100% sure whether those boilerplate html is needed or not.